### ISSUE TITLE
GitHub action build fails on changelog generation

### ISSUE THREAD

**Issue by piontec** [goreleaser/goreleaser#4052]
### What happened?

When running, `goreleaser` always fails on the "changlog" action, unless it is disabled, with this error:

```
 ⨯ release failed after 1m43s               error=open output/notes.md: no such file or directory
Error: The process '/opt/hostedtoolcache/goreleaser-action/1.18.2/x64/goreleaser' failed with exit code 1
```

### How can we reproduce this?

Here's a full run log and a public repo that can be used to replicate: https://github.com/piontec/kube-echo/actions/runs/5112513746/jobs/9190636079

### goreleaser version

```bash
1.18.2
```

### GoReleaser Check

- [X] goreleaser check shows no errors

### Search

- [X] I did search for other open and closed issues before opening this

### Supporter

- [ ] I am a [sponsor](https://github.com/sponsors/caarlos0/) of [Pro](https://goreleaser.com/pro) customer

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Additional context

I'm not a sponsor, but I found no instructions how to report a bug for the open source version.

**Comment by caarlos0** [comment]
> open output/notes.md: no such file or directory

the `--release-notes` is used to instruct a file to be used as input, and that file apparently does not exist.

tl;dr: working as expected.


> I'm not a sponsor, but I found no instructions how to report a bug for the open source version.

this is the correct one, that checkbox is just for me to prioritize sponsors et al, not mandatory to check it ;)

**Comment by piontec** [comment]
Thank you for your response!
I'm not using explicit `--release-notes` switch, so I was expecting `goreleaser` to generate the changelog completely automatically, from git log messages. I checked [docs here](https://goreleaser.com/customization/changelog/), and I find there is no mention of `notes.md` file nor what should it actually contain (and what will be generated). 
I've seen a mention about [custom release notes](https://goreleaser.com/customization/release/#custom-release-notes), but I don't want anything custom.

So, to be honest, I read the docs and I'm still not sure what I'm doing wrong and what should I do to fix this... I guess I'm getting something wrong about how the changelog generation works.

Ad. the "sponsor" checkbox, IIRC I had to check it, as it was marked mandatory - otherwise github didn't allow me to create the ticket from the bug report template.

**Comment by caarlos0** [comment]
Yes you are, line 49: https://github.com/piontec/kube-echo/blob/main/.github/workflows/release.yaml

About the checkbox, bug, will fix, thanks for reporting 🤘

**Comment by piontec** [comment]
Oh shit, I left it as a run arg from the action, and was checking only the goreleaser config! :( Sorry for being half-blind and thanks for help!

**Comment by caarlos0** [comment]
hehe, no problem, and thanks for the checkbox thingy heads up :D
